template/alpha_settings.py

```diff
@@ -21,8 +23,8 @@ class AlphaSettings(BaseAlphaSettings):
     alpha = AlphaFormula
     # trading target, including type and region (tuple(str, str))
     target = ('currency', 'TW')
-    # data used in the alpha (list(str))
-    data_list = ['currency_price_tw']
+    # data used in the alpha (tuple(str))
+    data_list = ('currency_price_tw',)
```

Comment:
I don't think this is a good idea.
While it is named `data_list` and it does make sense to be a list, we should make it to behave exactly like a list to our users.
We need it to be a tuple for engineering reasons, but that's not something users should care about.
We should build an comfortable environment for users, and such dirty work can be done internally.